ogc-rs/src/console.rs

```diff
@@ -0,0 +1,70 @@
+//! The ``console`` module of ``ogc-rs``.
+//!
+//! This module implements a safe wrapper around the console functions.
+
+use crate::mem_k0_to_k1;
+use std::ptr;
+
+struct Console(());
+
+impl Console {
+    /// Initializes the console subsystem with given parameters.
+    pub fn init(xstart: i32, ystart: i32, xres: i32, yres: i32, stride: i32) -> Console {
+        unsafe {
+            let framebuffer = mem_k0_to_k1(ogc_sys::SYS_AllocateFramebuffer(
```

Comment:
Separate the ``framebuffer`` into another variable to simplify ``CON_Init`` call.